lib/api/protocol/setContext.js

```diff
@@ -1,10 +1,40 @@
 const ProtocolAction = require('./_base-action.js');
 
 /**
- * Sets the context.
+ * Sets the context to be automated. Used when testing hybrid mobile apps using Appium.
  *
+ * More info here: https://appium.io/docs/en/commands/context/set-context/
+ *
+ * @example
+ * module.exports = {
+ *   'switch to webview context': function (browser) {
+ *     browser
```

Comment:
we should use an async testcase here.

Reply:
Didn't add an example with async/await since `waitUntil` is not working right now with async/await (raised issue https://github.com/nightwatchjs/nightwatch/issues/3511).

Should I add the example anyway?

Reply:
Also, since the commands inside the async callback to `waitUntil` are not working as expected (not returning a Promise), I'll make the existing examples to be async and then once the `waitUntil` issue is resolved, we can add proper examples `getContexts` command.